vue_app/public/index.html

```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width,initial-scale=1.0">
+    <title>Craig D'Arcy | Designer and Front-End Developer</title>
```

Comment:
would recommend handling this in-app (vue) if possible. There's this library `react-helmet` which is super useful for this kind of stuff, I'm sure there there's an analogue for vue out there!